05/index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="./styles/reset.css">
+    <link rel="stylesheet" href="./styles/global.css">
+    <link rel="stylesheet" href="./styles/mobile.css">
+    <link rel="stylesheet" href="./styles/desktop.css" media="(min-width: 800px)" >
+</head>
+<body>
+    <header class="row header">
+        <div class="container header__container">
+            <h1 class="header__logo cell"></h1>
+            <button class="hamburger"></button>
```

Comment:
Nie wiem czy tego buttona nie wrzuciłbym do `nav` bo to cześć nawigacji (ale nie jest to jakoś bardzo istotne)